--- fwupd-1.4.4/plugins/modem-manager/fu-qmi-pdc-updater.c.orig	2020-06-10 16:43:14.270352000 +0200
+++ fwupd-1.4.4/plugins/modem-manager/fu-qmi-pdc-updater.c	2020-07-07 06:24:20.714166201 +0200
@@ -226,11 +226,13 @@
 	guint		 token;
 } WriteContext;
 
+#if (QMI_MAJOR_VERSION < 1) || ((QMI_MAJOR_VERSION == 1) && (QMI_MINOR_VERSION < 26))
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wunused-function"
 G_DEFINE_AUTOPTR_CLEANUP_FUNC(QmiMessagePdcLoadConfigInput, qmi_message_pdc_load_config_input_unref)
 G_DEFINE_AUTOPTR_CLEANUP_FUNC(QmiMessagePdcLoadConfigOutput, qmi_message_pdc_load_config_output_unref)
 #pragma clang diagnostic pop
+#endif
 
 static void fu_qmi_pdc_updater_load_config (WriteContext *ctx);
 
@@ -435,11 +437,13 @@
 	guint		 token;
 } ActivateContext;
 
+#if (QMI_MAJOR_VERSION < 1) || ((QMI_MAJOR_VERSION == 1) && (QMI_MINOR_VERSION < 26))
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wunused-function"
 G_DEFINE_AUTOPTR_CLEANUP_FUNC(QmiMessagePdcActivateConfigInput, qmi_message_pdc_activate_config_input_unref)
 G_DEFINE_AUTOPTR_CLEANUP_FUNC(QmiMessagePdcActivateConfigOutput, qmi_message_pdc_activate_config_output_unref)
 #pragma clang diagnostic pop
+#endif
 
 static gboolean
 fu_qmi_pdc_updater_activate_config_timeout (gpointer user_data)
@@ -540,11 +544,13 @@
 					fu_qmi_pdc_updater_activate_config_ready, ctx);
 }
 
+#if (QMI_MAJOR_VERSION < 1) || ((QMI_MAJOR_VERSION == 1) && (QMI_MINOR_VERSION < 26))
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wunused-function"
 G_DEFINE_AUTOPTR_CLEANUP_FUNC(QmiMessagePdcSetSelectedConfigInput, qmi_message_pdc_set_selected_config_input_unref)
 G_DEFINE_AUTOPTR_CLEANUP_FUNC(QmiMessagePdcSetSelectedConfigOutput, qmi_message_pdc_set_selected_config_output_unref)
 #pragma clang diagnostic pop
+#endif
 
 static gboolean
 fu_qmi_pdc_updater_set_selected_config_timeout (gpointer user_data)
